core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/AbstractCassandraTest.java

```diff
@@ -77,6 +77,11 @@ public boolean getRemoteRouting()
             {
                 return true;
             }
+
+            @Override
+            public String getSerialConsistency(){
```

Comment:
Incorrect formatting {} should be placed on newlines.

Reply:
Done